rero_ils/modules/documents/templates/rero_ils/detailed_view_documents.html

```diff
@@ -1,5 +1,5 @@
 {%- extends config.RECORDS_UI_BASE_TEMPLATE %}
-{% set data = record.dumps() %}
+{% set document = record %}
```

Comment:
Why you don't use directly record on template ?

Reply:
It is exactly what I do. `record.dump()` is useless with `$ref` and instead of using a generic name `data`, I think it is better to use a more specific name variable i.e. `document`. Any other suggestion?